cli/reference/variables/metadata.md

```diff
@@ -32,6 +32,12 @@ The following keys are available in the `terramate.stack` object and can be acce
     - `relative` (string) The relative path of the stack from the repository root.
     - `to_root` (string) The relative path from the stack to the repository root (upwards).
 
+  ### Parent Stack Metadata
```

Comment:
We don't need a separate section for this. It should just be part of the stack metadata.

Just opened up a new bulletpoint appended to the list above

```
- `parent` (object) An object containing the metadata of the immediate parent stack if available
  - `id` (string) .....
```